### PR TITLE
DOC-1722: Edit descriptions for clusters/optimize index page

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1722-real/rs/clusters/optimize/

### CODE REVIEW

File: content/rs/clusters/optimize/_index.md [hunk 1]
```diff
@@ -10,4 +10,20 @@ aliases: [
...
+## [Benchmarking Redis Enterprise]({{< relref "/rs/clusters/optimize/memtier-benchmark" >}}) 
+Use the `memtier_benchmark` tool to perform a performance benchmark of Redis Enterprise Software.
+
+## [Disk sizing for heavy write scenarios]({{< relref "/rs/clusters/optimize/disk-sizing-heavy-write-scenarios" >}}) 
+Determine the required persistent disk space for heavy throughput databases.
+
+## [Cluster environment optimization]({{< relref "/rs/clusters/optimize/optimization" >}}) 
+Optimize your cluster environments for better performance.
+
+# [Turn off services to free system memory]({{< relref "/rs/clusters/optimize/turn-off-services" >}})
```
Comment (rrelledge) [inline content/rs/clusters/optimize/_index.md:22]:
```suggestion
## [Turn off services to free system memory]({{< relref "/rs/clusters/optimize/turn-off-services" >}}) 
```

File: content/rs/clusters/optimize/_index.md [hunk 2]
```diff
@@ -10,4 +10,20 @@ aliases: [
...
+## [Cluster environment optimization]({{< relref "/rs/clusters/optimize/optimization" >}}) 
+Optimize your cluster environments for better performance.
+
+# [Turn off services to free system memory]({{< relref "/rs/clusters/optimize/turn-off-services" >}}) 
+Turn off services to free memory and improve performance.
+
+## [Redis OSS Cluster API]({{< relref "/rs/clusters/optimize/oss-cluster-api" >}}) 
+Use the Redis OSS Cluster API improve performance and let applications stay current with cluster topology changes.
+
+# [WAIT command]({{< relref "/rs/clusters/optimize/wait" >}})
```
Comment (rrelledge) [inline content/rs/clusters/optimize/_index.md:28]:
```suggestion
## [WAIT command]({{< relref "/rs/clusters/optimize/wait" >}})
```

File: content/rs/clusters/optimize/_index.md [hunk 3]
```diff
@@ -10,4 +10,20 @@ aliases: [
...
+Determine the required persistent disk space for heavy throughput databases.
+
+## [Cluster environment optimization]({{< relref "/rs/clusters/optimize/optimization" >}}) 
+Optimize your cluster environments for better performance.
+
+# [Turn off services to free system memory]({{< relref "/rs/clusters/optimize/turn-off-services" >}}) 
+Turn off services to free memory and improve performance.
+
+## [Redis OSS Cluster API]({{< relref "/rs/clusters/optimize/oss-cluster-api" >}}) 
+Use the Redis OSS Cluster API improve performance and let applications stay current with cluster topology changes.
```
Comment (rrelledge) [inline content/rs/clusters/optimize/_index.md:26]:
```suggestion
Use the Redis OSS Cluster API to improve performance and keep applications current with cluster topology changes.
```

File: content/rs/clusters/optimize/_index.md [hunk 4]
```diff
@@ -10,4 +10,20 @@ aliases: [
...
+
+# [WAIT command]({{< relref "/rs/clusters/optimize/wait" >}})
+Use the WAIT command to take full advantage of Redis Enterprise Software's replication based durability.
```
Comment (rrelledge) [inline content/rs/clusters/optimize/_index.md:29]: Maybe use this other sentence from the article as the description?
```suggestion
Use the WAIT command to control the consistency and durability guarantees for the replicated and persisted database.
```

**Review by rrelledge — COMMENTED** [review]
Looks good aside from a few small typos and formatting issues.

**Review by lanceleonard — APPROVED** [review]
LGTM

**Pull request merged**